datafusion/physical-expr/src/aggregate/covariance.rs

```diff
@@ -319,281 +225,3 @@ impl Accumulator for CovarianceAccumulator {
         std::mem::size_of_val(self)
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
I was thinking whether we should move these tests into the new files rather than deleting them. 🤔

Reply:
you can move it to aggregate.slt like what in #10384 does

Reply:
you can create another PR for moving test

Reply:
Got it.